docs/usage/self-hosted-configuration.md

```diff
@@ -905,6 +905,22 @@ It also may mean that ignored directories like `node_modules` can be preserved a
 
 ## platform
 
+## platformVersion
```

Comment:
should we add `platformOptions` in both `configration-options.md` & `self-hosted-configuration.md`?

Reply:
yes, I think we need to.

Reply:
This will disturb the docs test, as we do not expect same option to be present in both `configuration-options.md` & `self-hosted-configuration.md`.

Should I add an exception in the tests, or something else?

Reply:
> Should I add an exception in the tests, or something else?

I don't like exceptions to tests. That makes it harder to understand what and why we're testing something.

> This will disturb the docs test, as we do not expect same option to be present in both `configuration-options.md` & `self-hosted-configuration.md`.

The test failure also shows that using the same name for a config option in two places is confusing.

Questions:

- Does `platformVersion` behave _exactly_ the same on Mend-hosted app and self-hosted?
- If there is a difference in behavior, should we use different names for each `platformVersion`? For example: `selfhostedPlatformVersion` and a plain `platformVersion`?